src/utils/getAccessTabData.ts

```diff
@@ -5,6 +5,21 @@ import { IAccessItem } from '../interfaces/searchResponse.interface';
 import { getOrganisationDetails } from './getOrganisationDetails';
 import { addSpaces, capitalizeWords } from './formatAggregationResponse';
 
+const rolePrecedence = ['owner', 'pointOfContact', 'custodian', 'distributor', 'originator'];
+
+const combineAndSortContacts = (contactForResource: any[], contact: any[]): any[] => {
```

Comment:
will this be any? can't we define a interface for this?

Reply:
Added interface for contact